ldk/javascript/readme.md

```diff
@@ -28,6 +28,43 @@ const merged = merge.merge(ldkConfig.default, {
 module.exports = merged;
 ```
 
+### Loop Permissions
+In order to ensure your Loop is executing in a secure manner, you must declare which network URLs, file system paths, and aptitudes your Loop will use.
+
+Permissions are declared inside of the Loop `package.json` within a `permissions` json object.
+
+```json
+  "permissions": {
+    "urlPermissions": ["google.com/*"],
+    "filesystemPermissions": ["/tmp"],
+    "aptitudePermissions": ["clipboard"]
```

Comment:
IMO stylistically, repeating the name property name under child properties is a bit clunky in Node/JS. I don't think this is necessarily the case in .Net.

I'm also wondering whether we want to plan in advance for future aptitude configurations. So instead of listing the aptitudes in this object, we could do something like this:

```yaml
permission:
  network:
    domains:
      - *.google.com
      - yahoo.ca   
  filesystem:
    paths:
      - /tmp/**/*
      - /usr/local/bin
  clipboard: true # Includes text only
```

And then if we decide we want to include clipboard images, we can then say:
```yaml
permission:
  clipboard:
    text: true
    image: true
```

Reply:
I think we should also consider which aptitudes are enabled automatically, like Whispers.

Reply:
I think this totally makes sense, organizing things this way. I think we should be as specific as possible with the keys. So like you did with network `domains`, that is good. But for files, maybe it should be `pathGlobs`. Just in case we ever allow file `pathRegexes`. And same for the chance for network `urlGlobs` or something.

I do wonder if do something like: ```
"permissions": {
  "network": {whatever...},
  "clipboard: {},
}
```
would make sense just to do for enabling aptitudes that don't currently have sub-properties. Just trying to set us up for the long-term and potential for backwards-compatibility, so we don't have to expect a boolean now but an object later. And the way we check for aptitude access is just the existence of the aptitude name in the sub-keys.